modules/wp-rest/classes/wp-post.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Elementor\Modules\WpRest\Classes;
+
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Core\Utils\Collection;
+use Elementor\Modules\GlobalClasses\Utils\Error_Builder;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class WP_Post {
+	const MAX_COUNT = 100;
+	const NAMESPACE = 'elementor/v1';
+	const ENDPOINT = 'post';
+
+	const EXCLUDED_POST_TYPES_KEY = 'excluded_post_types';
+	const TERM_KEY = 'term';
+	const KEYS_FORMAT_MAP_KEY = 'keys_format_map';
+	const MAX_COUNT_KEY = 'max_count';
+	const NONCE_KEY = 'x_wp_nonce';
+
+	private ?Wordpress_Adapter_Interface $wp_adapter = null;
+
+	public function __construct( ?Wordpress_Adapter_Interface $wp_adapter = null ) {
+		$this->wp_adapter = $wp_adapter ?? new Wordpress_Adapter();
+	}
+
+	public function register( bool $override_existing_endpoints = false ): void {
+		register_rest_route( self::NAMESPACE, self::ENDPOINT, [
+			[
+				'methods' => \WP_REST_Server::READABLE,
+				'permission_callback' => fn ( \WP_REST_Request $request ) => $this->validate_access_permission( $request ),
+				'args' => $this->get_args(),
+				'sanitize_callback' => 'esc_attr',
+				'callback' => fn ( \WP_REST_Request $request ) => $this->route_wrapper( [ $this, 'get_posts' ], $request ),
```

Comment:
```suggestion
				'callback' => fn ( \WP_REST_Request $request ) => $this->route_wrapper( fn() => $this->get_posts( $request ) ),
```

Just for consistency